financialaid/admin.py

```diff
@@ -0,0 +1,57 @@
+"""
+Admin views for Financial Aid app
+"""
+
+from django.contrib import admin
+
+from financialaid.models import (
+    Coupons,
+    Documents,
+    FinancialAid,
+    FinancialAidAudit,
+    TierProgram,
+    Tiers,
+)
+
+
+class CouponsAdmin(admin.ModelAdmin):
+    """Admin for Coupons"""
+    model = Coupons
+
+
+class DocumentsAdmin(admin.ModelAdmin):
+    """Admin for Documents"""
+    model = Documents
+
+
+class FinancialAidAdmin(admin.ModelAdmin):
```

Comment:
I agree: I put it now just because it is probably useful while we write the code. We can remove it when we realize what goes in this table.